clang/lib/CodeGen/CGClass.cpp

```diff
@@ -246,7 +246,9 @@ ApplyNonVirtualAndVirtualOffset(CodeGenFunction &CGF, Address addr,
 
   // Apply the base offset.
   llvm::Value *ptr = addr.getPointer();
-  ptr = CGF.Builder.CreateBitCast(ptr, CGF.Int8PtrTy);
+  llvm::Type *ResTy = llvm::PointerType::getInt8PtrTy(
```

Comment:
`ResTy` name is not consistent with the rest of the names in this function.

Reply:
fixed